src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,64 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import { useSearchParams, Link } from 'react-router-dom';
+import cn from 'classnames';
+import { SexFilter } from '../types';
+import { getSearchWith } from '../utils/getSearchWith';
+
+type Props = {};
+
+const sexParams = [
+  { sortBy: SexFilter.All },
+  { sortBy: SexFilter.Male },
+  { sortBy: SexFilter.Female },
+];
+
+const centuriesParams = [
+  { sortBy: '16' },
+  { sortBy: '17' },
+  { sortBy: '18' },
+  { sortBy: '19' },
+  { sortBy: '20' },
+];
+
+export const PeopleFilters: React.FC<Props> = ({}) => {
```

Comment:
You have destructured an empty object in your function parameters. If you do not expect any props, you can remove the empty object.